src/Sulu/Bundle/AdminBundle/Command/ViewDebugCommand.php

```diff
@@ -33,7 +33,7 @@
 /**
  * @internal
  */
-#[AsCommand(name: 'sulu:debug:admin:views', description: 'Display current Views for the admin')]
+#[AsCommand(name: 'sulu:admin:debug-views', description: 'Display current Views for the admin')]
```

Comment:
In the doc it is called `debug-view` shold we go with that?

https://github.com/sulu/sulu-docs/pull/793/files#diff-81b84d4fcee0a41917f5da6e64293

![Bildschirmfoto 2024-04-25 um 18 30 49](https://github.com/sulu/sulu/assets/1698337/a1fda25a-ebe0-4d40-bc75-b76f20855181)
99ea991108e2525d7b3c1ed4592031a656fR424-R431

Reply:
Oh, good catch I've completely overlooked that 🙈  personally I would go with singular, but I don't have a strong opinion

Reply:
Lets go with the singular then